src/include/cpp-btree/btree.h

```diff
@@ -55,6 +55,7 @@
 #include <iterator>
 #include <limits>
 #include <new>
+#include <ostream>
```

Comment:
if we really want to have `operator<<` for btree. could just remove `iosfwd` couple lines above as `ostream` has the complete type definition for ostream. while iosfwd just offers the forward declaration for it.

Reply:
@tchaikov FreeBSD seems happy with that.